tests/conftest.py

```diff
@@ -212,13 +213,66 @@ def numeric_high_card_dataset(test_backend, numeric_high_card_dict):
     return get_dataset(test_backend, numeric_high_card_dict, schemas=schemas)
 
 
+@pytest.fixture
+def datetime_dataset(test_backend):
+    data = {
+        "datetime": [
+            str(datetime.datetime(2020, 2, 4, 22, 12, 5, 943152)),
```

Comment:
Why are these being converted to string first?

Reply:
In the get_dataset util function, column values are converted/parsed to datetime. For all the backends except spark, pd.to_datetime is used, which doesn't complain if values are already datetime, but spark uses parse(val), which barfs if val isn't a string.